pkg/apicapi/apicapi.go

```diff
@@ -454,7 +459,7 @@ loop:
 		select {
 		case <-syncHook:
 			conn.fullSync()
-		case <-refreshTicker.C:
+		case <-refreshTickerChan:
```

Comment:
Thanks for updating this part, I totally missed it. So thinking about this a little more, the refresh will never happen if conn.RefreshInterval is set to zero, right? I think we need to revert to using the original value when the config value is set to zero.